pkg/sql/types/alias.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2015 The Cockroach Authors.
```

Comment:
2020

Reply:
nah mate i'm pretty sure it's 2015, no corona